src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/Tensor.cs

```diff
@@ -1225,7 +1225,6 @@ public string GetArrayString(bool includeWhitespace = true)
             var indices = new int[Rank];
             var innerDimension = Rank - 1;
             var innerLength = dimensions[innerDimension];
-            var outerLength = Length / innerLength;
```

Comment:
I assume innerLength can't be 0 and that the code isn't relying on that for an exception, but it'd be good to validate.  @tannergooding, removing this is ok, correct?

Reply:
> I assume innerLength can't be 0

It can't be 0. The constructor has this check:

https://github.com/dotnet/runtime/blob/5b561b16e0c8000789af48438a9d1f144de32c3a/src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/Tensor.cs#L293-L296